user_scripts/check_run_stats.sh

```diff
@@ -0,0 +1,61 @@
+#################################################
```

Comment:
I'm assuming this is a subset of the variant_calling.sh script (only doing the stats part, for people who don't want to commit to a week and a half of variant calling but still want to get the basic run stats that variant calling performs at the beginning)